### DISCUSSION TITLE
Issue with installing new apps (can't find them on the app launcher) unless I restart

### DISCUSSION THREAD

**Discussion by Raikuyami** [end-4/dots-hyprland#910] [Q&A]
Hii, I just recently installed this pre-made hyprland config which I really like, however, when I try to install some programs (like obsidian, vesktop, youtube-music-for-desktop) or anything else, I can't find them immediatly on the app launcher unless I restart the whole computer. Is there a way or any command that I can use to just have them working after I install them ? 

As a temporary, I just launch them from the terminal, but I wouldn't want to restart my computer everytime I add something new.

Thanks !

**Comment by meflove** [comment]
Ctrl+Super/win+R

**Reply by kvazar-tsar** [reply]
After Ctrl+Super/win+R there are not displayed programs that were installed this session. Literally need to reset for them to appear in app launcher. I just installed firefox and after opening app launcher it doesnt show it yet it can be run form terminal.

**Comment by PharmaRacist** [comment]
update the dotfiles i think he added a watcher now it doesnt require this

**Comment by RAZUMNO** [comment]
This issue still persist. Would be nice if there was a way to auto-update installed/deleted software in app launcher

**Reply by N1ckstars** [reply]
yeah this is still an issue unfortunately, it would be nice if there wa s away to restart the launcher somehow but i dont even know what the launcher is called, it seems entirely custom

**Reply by PharmaRacist** [reply]
Just making a watcher for the apps dir
